.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -270,8 +330,20 @@ jobs:
             cat /usr/local/lib/igc/IGCTAG.txt
           fi
 
+    - name: Download E2E Binaries
+      if: inputs.e2e_binaries_artifact != '' && github.event_name != 'workflow_run'
+      uses: actions/download-artifact@v4
+      with:
+        name: ${{ inputs.e2e_binaries_artifact }}
+    - name: Extract/Setup E2E Binaries
+      if: inputs.e2e_binaries_artifact != ''
+      run: |
+        mkdir build-e2e
+        tar -I '${{ inputs.e2e_binaries_decompress_command }}' -xf ${{ inputs.e2e_binaries_archive }} -C build-e2e
+        rm -f ${{ inputs.e2e_binaries_artifact }}
```

Comment:
is this just to save disk space?

Reply:
I'm not certain, but I think so. I did this to mirror the `Extract/Setup SYCL toolchain` step that we already have. I can remove it if that would be preferable.

Reply:
Removed this and it seems to work fine still